front50-core/src/main/groovy/com/netflix/spinnaker/front50/model/pluginartifact/PluginArtifact.java

```diff
@@ -0,0 +1,129 @@
+/*
+ * Copyright 2019 Netflix, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.netflix.spinnaker.front50.model.pluginartifact;
+
+import com.fasterxml.jackson.annotation.JsonIgnore;
+import com.netflix.spinnaker.front50.model.Timestamped;
+import java.time.Instant;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Optional;
+import javax.annotation.Nonnull;
+import lombok.Data;
+
+/**
+ * A Spinnaker plugin's release artifact metadata.
+ *
+ * <p>This model is used internally by Spinnaker to track what plugins are available for services to
+ * install, as well as the specific releases that should be installed.
+ */
```

Comment:
New entity requires Liquibase migration.?

Reply:
Yep - added that now.